website/content/en/docs/cli/operator-sdk.md

```diff
@@ -0,0 +1,31 @@
+## operator-sdk
```

Comment:
These CLI docs are auto-generated. Should we update the make target so that this directory gets updated as part of that generation?

Reply:
Once this merges, I'm going to put the organizational framework in place. From there, each section can be done in parallel, so CLI updates will be a separate ticket.